openapi/scim/scim.yaml

```diff
@@ -678,16 +680,20 @@ components:
           example: Hollywood
         region:
           type: string
-          example: CA
+          pattern: countrycode-statecode
```

Comment:
This should be a regex pattern. You can copy from the screenshot/ platform users API.